src/implementations/config/create.ts

```diff
@@ -0,0 +1,192 @@
+import {
+  Issuer,
+  RevocationType,
+  IdentityProofType,
+  OpenAttestationDocument,
+} from "@govtechsg/open-attestation/dist/types/__generated__/schema.2.0";
+import fs from "fs";
+import path from "path";
+import { info } from "signale";
+import { readFile } from "../../implementations/utils/disk";
+import tradetrustConfig from "./templates/tradetrust.json";
+import { handler as createTemporaryDns } from "../../commands/dns/txt-record/create";
+import { CreateConfigCommand } from "../../commands/config/config.type";
+import { getDocumentStoreAddress } from "./documentStore";
+import { getTokenRegistryAddress } from "./tokenRegistry";
+
+interface ConfigFile {
+  wallet: EncryptedWallet;
+  forms: Form[];
+}
+
+interface EncryptedWallet {
+  type: "ENCRYPTED_JSON";
+  encryptedJson: string;
+}
+
+interface Form {
+  type: "VERIFIABLE_DOCUMENT" | "TRANSFERABLE_RECORD";
+  defaults: OpenAttestationDocument;
+}
+
+interface TypesOfForms {
+  type: "VERIFIABLE_DOCUMENT" | "TRANSFERABLE_RECORD";
+  identityProofTypes: (IdentityProofType | undefined)[];
+}
+
+const sandboxEndpointUrl = "https://sandbox.fyntech.io";
+
+export const create = async ({
+  encryptedWalletPath,
+  outputDir,
+  configType,
+  configTemplatePath,
+}: CreateConfigCommand): Promise<string | undefined> => {
+  try {
+    const wallet = await readFile(encryptedWalletPath);
+    const walletObject = JSON.parse(wallet);
+
+    info(`Wallet detected at ${encryptedWalletPath}`);
+
+    const configFile: ConfigFile = await getConfigFile(configType, configTemplatePath);
+
+    const { forms } = configFile;
+
+    const getContractAddress = async (
+      typesOfForms: TypesOfForms[],
+      formType: "VERIFIABLE_DOCUMENT" | "TRANSFERABLE_RECORD",
+      identityProofType: "DNS-DID" | "DNS-TXT"
+    ): Promise<string> => {
+      const isValidForm = typesOfForms.some(
+        (item: TypesOfForms) =>
+          item.type === formType && (<any>Object).values(item.identityProofTypes).includes(identityProofType)
+      );
+
```

Comment:
just throw early

`throw new Error("Invalid form detected in config file, please update the form before proceeding.");`

Reply:
ok